components/agentSearchForm/index.jsx

```diff
@@ -30,30 +30,46 @@ import {
   SimpleInput,
 } from "@inrupt/prism-react-components";
 import { useSession } from "@inrupt/solid-ui-react";
+import { isPodOwner } from "../../src/solidClientHelpers/utils";
 
 export const TESTCAFE_ID_ADD_AGENT_BUTTON = "add-agent-button";
 
+const AGENT_TYPE_MAP = {
+  contacts: {
```

Comment:
Just a style question - should `contacts` and `permissions` be capitalized (or the actual types camelcase)? AGENT_TYPE_MAP.contacts.OWN_WEBID_ERROR_MESSAGE feels a little awkward.

Reply:
@ajacksified I spoke too soon. It does look goofy when you write it out like that but that's not how we're using it. See https://github.com/inrupt/pod-browser/blob/f32831cfe711796b7ba2d79b0d81d39376964e68/components/agentSearchForm/index.jsx#L134